roles/ovirt-vm-infra/README.md

```diff
@@ -60,27 +68,24 @@ The item in `disks` list of `profile` dictionary can contain following attribute
 | name               | UNDEF          | The name of the additional disk.  |
 | storage_domain     | UNDEF          | The name of storage domain where disk should be created. |
 | interface          | UNDEF          | The interface of the disk. |
+| format             | UNDEF          | Specify format of the disk.  <ul><li>cow - format is used, disk will by created as sparse, so space will be allocated for the volume as needed, also known as thin provision</li><li>raw - format is used, disk storage will be allocated right away, also known as preallocated.</li></ul> |
```

Comment:
<ul><li>cow - If set, the disk will by created as sparse disk, so space will be allocated for the volume as needed. This format is also known as thin provisioned disks</li><li>raw - If set, disk space will be allocated right away. This format is also known as preallocated disks.</li></ul>

Reply:
done